docs/src/main/asciidoc/mp/testing/testing-ng.adoc

```diff
@@ -69,6 +69,14 @@ I.e. By default, the test instance is re-used between test methods.
 
 NOTE: The test instance is not re-used between CDI container, using a dedicated CDI container implies a new test instance
 
+=== Using meta-annotations
+`@HelidonTest` can be used in a meta-annotation:
```

Comment:
Meta annotations should work for methods as well.

I don't see any test for it (under microprofile/tests/testing), can you double check if it works ?
If it does work, please add a test for it.

We could have two examples, the one you made, and another one like this:

```java
@Test
@AddBean(FirstBean.class)
@AddBean(SecondBean.class)
@DisableDiscovery
@Target(ElementType.METHOD)
@Retention(RetentionPolicy.RUNTIME)
public @interface MyTestMethod {
}

@HelidonTest
class MyTest {

    @MyTestMethod
    void testOne() {
    }

    @MyTestMethod
    void testTwo() {
    }
}
```

Reply:
It passed for me, it could be something environmental that alters the ordering.
What JDK are you using ?

Reply:
OpenJDK Runtime Environment Temurin-21.0.2+13 (build 21.0.2+13-LTS)